core/templates/pages/exploration-editor-page/feedback-tab/feedback-tab.component.html

```diff
@@ -91,49 +93,47 @@ <h2>Exploration Feedback</h2>
       </div>
     </div>
 
-
     <hr>
 
-    <div class="row" ng-if="$ctrl.userIsLoggedIn">
+    <div class="row" *ngIf="userIsLoggedIn">
       <div class="col-lg-12 col-md-12 col-sm-12">
         <div>
           <label for="tmpMessageText">
             Add new message
-            <span ng-if="$ctrl.isExplorationEditable() && !$ctrl.activeThread.isSuggestionThread()">(and/or change status)</span>
+            <span *ngIf="isExplorationEditable() && !activeThread.isSuggestionThread()">(and/or change status)</span>
           </label>
-          <textarea class="form-control e2e-test-feedback-response-textarea oppia-autofocus" ng-model="$ctrl.tmpMessage.text"
-                    id="$ctrl.tmpMessageText" rows="6"
-                    ng-disabled="$ctrl.messageSendingInProgress" focus-on="tmpMessageText">
+          <textarea class="form-control e2e-test-feedback-response-textarea oppia-autofocus" [(ngModel)]="feedbackMessage.text"
+                    id="tmpMessageText" rows="6"
+                    [disabled]="messageSendingInProgress" (focus)="tmpMessageText">
```

Comment:
Should we update the occurences of "tmpMessageText" to match the new variable feedbackMessage?

Reply:
done